Auth0/SafariWebAuth.swift

```diff
@@ -76,6 +76,20 @@ class SafariWebAuth: WebAuth {
         return self
     }
 
+    func responseType(_ responseType: [ResponseType]) -> Self {
```

Comment:
We need to enforce that a response_type is picked here and by default should include `code`